atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/ClusterMetadataUtils.java

```diff
@@ -45,6 +49,15 @@ private ClusterMetadataUtils() {
         // util class
     }
 
+    public static TableMetadata getTableMetadata(Metadata metadata, String keyspace, String table) {
+        KeyspaceMetadata keyspaceMetadata = metadata.getKeyspace(keyspace);
+        Optional<TableMetadata> maybeTable = keyspaceMetadata.getTables().stream()
+                .filter(tableMetadata -> tableMetadata.getName().equals(table))
+                .collect(MoreCollectors.toOptional());
+        return maybeTable.orElseThrow(() -> new SafeIllegalArgumentException(
+                "Can't find table", SafeArg.of("keyspace", keyspace), SafeArg.of("table", table)));
```

Comment:
how do you know that the table name is safe?

Reply:
copied code, and so far this is only used for internal tables, but you're right that I should be replacing with LoggingArgs.tableRef here.